core/validation/pre_transformation_validate.py

```diff
@@ -0,0 +1,148 @@
+    """
+    form_version, reporting_period = GET_FORM_VERSION_AND_REPORTING_PERIOD[reporting_round]
+
+    wrong_input_checks = {}
+    sheet1 = workbook.get("1 - Start Here")
+    sheet2 = workbook.get("2 - Project Admin")
+
+    wrong_input_checks["Form Version"] = (
+        sheet1.iloc[6][1],
+        (form_version,),
+    )
+    wrong_input_checks["Reporting Period"] = (sheet1.iloc[4][1], (reporting_period,))
+    wrong_input_checks["Fund Type"] = (
+        FUND_TYPE_TO_TD_OR_HS.get(sheet2.iloc[5][4], sheet2.iloc[5][4]),
+        ("TD", "HS"),
+    )
+    wrong_input_checks["Place Name"] = (
+        str(sheet2.iloc[6][4]).strip(),
+        tuple(TF_PLACE_NAMES_TO_ORGANISATIONS.keys()),
+    )
+
+    failures = check_values(wrong_input_checks, "wrong_input_failure")
+
+    if failures:
+        raise ValidationError(validation_failures=failures)
+
+
+def conflicting_input_validation(workbook: dict[str, pd.DataFrame]):
+    """
+    Validate conflicting input in the given workbook.
+
+    This function checks for conflicting input in the submitted workbook.
+
+    :param workbook: A dictionary containing sheets of the workbook.
+    :raises ValidationError: If conflicting input validation fails, raises a ValidationError with associated failures.
+    """
+    conflicting_input_checks = {}
+    sheet = workbook.get("2 - Project Admin")
+
+    conflicting_input_checks["Place Name vs Fund Type"] = (
+        FUND_TYPE_TO_TD_OR_HS.get(sheet.iloc[5][4]),
+        tuple(PLACE_TO_FUND_TYPE.get(str(sheet.iloc[6][4]).strip())),
+    )
+
+    failures = check_values(conflicting_input_checks, "wrong_input_failure")
+
+    if failures:
+        raise ValidationError(validation_failures=failures)
+
+
+def check_values(
+    failure_dictionary: dict, failure_type: str
+) -> list[vf.WrongInputFailure] | list[vf.UnauthorisedSubmissionFailure] | None:
+    """
+    Iterates through a dictionary of the relevant form input, and returns a list of specified errors for every
+    mismatch between actual and expected input.
+
+    :param failure_dictionary: A dictionary containing information for failure checks.
+    :param failure_type: A string representation of the type of failure.
+    :return: A list of ValidationFailure objects representing the failures, if any, or None if no errors are found.
+    """
+    str_repr_of_failure_to_actual_failure = {
+        "wrong_input_failure": vf.WrongInputFailure,
+        "unauthorised_submission_failure": vf.UnauthorisedSubmissionFailure,
+    }
+    failure = str_repr_of_failure_to_actual_failure[failure_type]
+    failures = []
+
+    for value_descriptor, (entered_value, expected_values) in failure_dictionary.items():
+        if entered_value not in expected_values:
+            failures.append(
+                failure(value_descriptor=value_descriptor, entered_value=entered_value, expected_values=expected_values)
+            )
+
+    return failures
```

Comment:
This is a similar function to what I expect the config described above would be used in